packages/test/setupTests.ts

```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+import ResizeObserver from 'resize-observer-polyfill';
+
+global.ResizeObserver = ResizeObserver;
```

Comment:
`ResizeObserver` semble bien supporté https://caniuse.com/?search=ResizeObserver pourquoi ajouter un polyfill ?

Reply:
Une erreur de jest que je n'arrivais pas à résoudre jusqu'à trouver cette [Issue](https://github.com/ZeeCoder/use-resize-observer/issues/40) qui indique le fix.
Sans le polyfill, Jest lançait des erreur `ResizeObserver is not defined`